### DISCUSSION TITLE
Combined two conditions in scATAC+scRNAseq

### DISCUSSION THREAD

**Discussion by biobarani** [stuart-lab/signac#1033] [Q&A] [answered]
Hello Signac members,

I just wonder how to perform the following analysis, since I was asked to perform cluster wise variation in DA peaks. 

So How to combine two conditions (KO+WT) and get cluster-wise Differentially Accessible (DA)peaks between two different conditions (For example cluster1 (KO) vs. cluster1(WT)?

Your help is really appreciated. Thanks in advance.

**Comment by timoast** [comment, marked as answer]
You can find an example of how to do this in the [Seurat vignette](https://satijalab.org/seurat/articles/integration_introduction.html)
